### DISCUSSION TITLE
POEditor provider?

### DISCUSSION THREAD

**Discussion by freiondrej-lmc** [symfony/symfony#44905] [Q&A] [answered]
Hello, is there a poeditor provider package? I only found https://github.com/symfony/symfony/issues/41374, that it was meant to be implemented as part of symfony, but that there was a roadblock and I could not find any further progress on the topic. Are there any alternatives?
Thank you! 🙂 Symfony rocks 🚀

**Comment by stof** [comment, marked as answer]
The poeditor provider was reverted from the 5.3 release because it was not working due to POEditor using the xliff files in a weird way. Until this is fixed on their side (allowing us to use our existing XliffFileLoader), we won't implement it in the core.

**Comment by chargelenni** [comment]
Any news here?

**Reply by MatTheCat** [reply]
See https://github.com/symfony/symfony/issues/41374#issuecomment-2585290248
